src/Components/Auction/__stories__/AuctionRegistrationModal.story.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import { storiesOf } from "storybook/storiesOf"
+import { AuctionRegistrationModal } from "../AuctionRegistrationModal"
+
+const submitHandler = p => {
```

Comment:
Nitpick, but I generally avoid single letter variables as it can sometimes be difficult to guess what they stand for. What do you think of expanding this to `props`?